tests/server-multiple-retention-policies/docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: '3.3'
+services:
+
+  ldes-server:
+    container_name: ${USECASE_NAME:-basic-retention}-ldes-server
+    image: ghcr.io/informatievlaanderen/ldes-server:${LDES_SERVER:-latest}
+    environment:
+      - MONGOCK_MIGRATIONSCANPACKAGE_0=be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset1
+      - MANAGEMENT_TRACING_ENABLED=False
+      - SIS_DATA=/tmp
+      - SPRING_DATA_MONGODB_DATABASE=gipod
+      - SPRING_DATA_MONGODB_HOST=ldes-mongodb
+      - SPRING_DATA_MONGODB_PORT=27017
+      - LDES-SERVER_HOST-NAME=http://localhost:${LDES_SERVER_PORT:-8080}
+    volumes:
+      - ../../data/epsg:/tmp/Databases:rw
+    ports:
+      - ${LDES_SERVER_PORT:-8080}:8080
+    networks:
+      - ldes
+    depends_on:
+      - ldes-mongodb
+
+  ldes-message-generator:
+    container_name: ${USECASE_NAME:-basic-retention}_member-generator
+    image: ghcr.io/informatievlaanderen/test-message-generator:${MEMBER_GENERATOR_TAG:-latest}
+    environment:
+      - TEMPLATE=[{"@id":"http://example.org/id{{index}}","@type":["http://example.org/id"],"http://example.org/modifiedAt":[{"@value":"{{timestamp}}","@type":"http://www.w3.org/2001/XMLSchema#dateTime"}],"http://example.org/isVersionOf":[{"@value":"version"}]}]
+      - TARGETURL=http://basic-retention-ldes-server:${LDES_SERVER_PORT:-8080}/mobility-hindrances
+      - MIMETYPE=application/json-ld
+    networks:
+      - ldes
+    profiles:
+      - delayed-started
+
+  ldes-message-generator-2:
```

Comment:
I wonder if you need 2 message generators? Couldn't you set the template as environment variable, run it, stop it, change environment variable, and start again?

Then you could use the standard name for this container test-message-generator, which is in line with all the other tests and will be easier for automation.